README.md

```diff
@@ -45,4 +45,6 @@ shared publicly (we will ask for consent before recording).
 
 ## License
 
-CC0 1.0 Universal, see [`LICENSE`](./LICENSE) file.
+Specification: CC0 1.0 Universal, see [`LICENSE`](./LICENSE) file.
+
+Reference implementation: MIT, see [`opnode/LICENSE`](./opnode/LICENSE) file.
```

Comment:
What about contracts?

Reply:
Good clarification point! Smart contract sources have been licensed as `MIT` in the main optimism monorepo, and should be thought of as part of the reference software, so these should also be `MIT` licensed.